litmus-portal/frontend/src/pages/Community/styles.ts

```diff
@@ -30,36 +30,45 @@ const useStyles = makeStyles((theme) => ({
       flexDirection: 'column',
     },
   },
-
   header2: {
     fontSize: '1.5rem',
-    fontWeight: 'bold',
+    fontWeight: 'normal',
     paddingTop: theme.spacing(2),
+    paddingBottom: theme.spacing(1),
```

Comment:
can we use single margin property ?

Reply:
No. Changes here are in accordance with the design. Please refer to https://www.geeksforgeeks.org/css-padding-vs-margin/ for more info

Reply:
i meant single padding property. please update @whitetiger1399